episodes/08-quarto.qmd

```diff
@@ -0,0 +1,247 @@
+---
+title: Getting started with Quarto (Optional)
+teaching: 30
+exercises: 15
+format:
+  html:
+    df-print: paged
+    keep-md: true
+source: md
+---
+
+```{r}
+#| label: setup
+#| include: false
+source("data/download_data.R")
+```
+
+::::::::::::::::::::::::::::::::::::::: objectives
+
+- Create a .qmd document containing R code, text, and plots
+- Create a YAML header to control output
+- Understand basic syntax of Quarto and Markdown
+- Customise code chunks to control formatting
+- Use code chunks and in-line code to create dynamic, reproducible documents
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::: questions
+
+- What is Quarto?
+- How can I integrate my R code with text and plots?
+- How can I convert .qmd files to .html?
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Creating a Quarto file
+
+To create a new Quarto document in RStudio, click File -> New File ->
+Quarto Document:
+
+![](fig/quarto-new.png){alt="Screenshot of the New Quarto file dialogue box in RStudio"}
+
+Then click on 'Create Empty Document'. Normally you could enter the title of
+your document, your name (Author), and select the type of output, but we will be
+learning how to start from a blank document.
+
+## Basic components of Quarto
+
+To control the output, a YAML (YAML Ain't Markup Language) header is needed:
+
+```
+---
+title: "My Awesome Report"
+author: "Emmet Brickowski"
+date: ""
+format: html
+---
+```
+
+The header is defined by the three hyphens at the beginning (`---`) and the
+three hyphens at the end (`---`).
+
+Although not recommended, you can leave the YAML out. Then the output will be 
+by default a HTML file. It's still better to include the file format in the YAML 
+header by adding the line `format: html`. You can also adapt the `format` of
+the file, to `pdf` or `docx`. We will start with an HTML document and discuss the other
+options later.
+
+You can add some additional information about your document in the YAML header such as `title`, `date` and `author`. This information will be displayed at the top of your document.  There are many more fields that can be added to the YAML header that provide additional information about the document or define the behaviour of the file. But we won't discuss them now. 
```

Comment:
```suggestion
You can add more information about your document in the YAML header such as `title`, `date` and `author`. This information will be displayed at the top of your document.  There are many more fields that can be added to the YAML header that provide additional information about the document or define the behaviour of the file. But we won't discuss them now. 
```

Reply:
In the write-up we say it's not compulsory to have any of this information but you can add it. It's directly derived from the Rmarkdown lesson. I think other options also differ by output format. I would add a link to the full list format options here for reference.